core/src/main/java/com/linecorp/armeria/server/auth/AuthServiceBuilder.java

```diff
@@ -153,17 +156,26 @@ public AuthServiceBuilder onFailure(AuthFailureHandler failureHandler) {
         return this;
     }
 
+    /**
+     * Sets the {@link MeterRegistry} that collects stats.
+     * If unspecified, a default one is used.
+     */
+    public AuthServiceBuilder meterRegistry(MeterRegistry meterRegistry) {
```

Comment:
We don't need this because we can get the sever's current `MeterRegistry` from `ServiceConfig`. Please check my comment on `AuthService.java`.

Reply:
Thank you! 
I tried to refactor based on your feedback. 
Could you PTAL again? 🙇